pypit/settings/settings.lris_blue

```diff
@@ -118,7 +118,7 @@ keyword dispangle None                 # Tilt
 
 ### Fits properties
 fits timeunit mjd                   # The unit of keyword+time (s=seconds, m=minutes, h=hours, or any of the astropy Time formats)
-fits calwin 12.0                    # The window of time in hours to search for calibration frames for a science frame
+fits calwin  0                      # The window of time in hours to search for calibration frames for a science frame
```

Comment:
I'm not sure I understand what having calwin=0 hours does here? Will PYPIT automatically use the supplied calibration frames for reduction instead of checking that the window of time is satisfied?